bin/jobsub

```diff
@@ -1,64 +1,68 @@
 #!/usr/bin/python3 -I
+import argparse
 import os
+import os.path
+import sys
 
-with os.popen(os.environ.get("PAGER", "less"), "w") as f:
-    print(
-        """
-Available jobsub_lite commands are:
+if os.environ.get("LD_LIBRARY_PATH", ""):
+    os.environ["HIDE_LD_LIBRARY_PATH"] = os.environ["LD_LIBRARY_PATH"]
+    del os.environ["LD_LIBRARY_PATH"]
+    os.execv(sys.argv[0], sys.argv)
 
-    jobsub_submit [-G experiment] [options] script-url [script arguments]
-        submit jobs/dags to the Grid
-    jobsub_q [-G experiment] [options]
-        show queued/running jobs
-    jobsub_fetchlog [-G experiment] [options] jobid ...
-         fetch job output logs
-    jobsub_history
-         show completed jobs
-    jobsub_hold
-    jobsub_release
-    jobsub_rm
-         Mangage job queue/abort jobs
-    decode_token.sh
-         Print JSON dump of token files
+PREFIX = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-    All josub_lite commands will print usage with --help, and
-    "man jobsub_..." will give a manual page, or see:
-    https://fifewiki.fnal.gov/wiki/Jobsub_Lite#User_Documentation
+sys.path.append(os.path.join(PREFIX, "lib"))
 
-jobsub_lite also wraps the following condor commands to accept '-G group':
+# pylint: disable=wrong-import-position, ungrouped-imports
```

Comment:
pylint is complaining because in all the other executables, we ignore `import-error` at this point.  Could you please add `import-error` to this list?  I also believe that you can't have a space between checks to ignore, so the whole line should read:

`# pylint: disable=wrong-import-position,ungrouped-imports,import-error`

This should fix the pylint checks.

Reply:
Done.   Its weird though, I have the pre-commit stuff turned on and it was passing pylint for me...  Maybe I have an old version?  Sigh. 
I'm also still seeing test_get_schedd_1 fail, it keeps getting jobsu01 instead of jobsubdevgpvm01 (?!) 
And test_condor_submit_dag1 is failing for me: well the test runs, but its output doesn't show up, so the failure shows up in check_output... 
If I look interactively, jobsub_fetchlog for its job doesn't get any output, but jobsub_fetchlog -condor does; so
something hosey in getting the tokens to the job(?).   I mean, it doesn't run long enough for them to expire or
anything.

Reply:
Hm... I'm wondering if it is because the .cmd file has a hardcoded, old, `+JobsubOutputURL=` line... perhaps
one of us owns that directory and the other can't put stuff there(?)...   I will see about tweaking that...

Reply:
For the test_get_schedd_1 failure, what condor_status command is being run?

Reply:
It uses the python library Collector class to do a query... 
`IsJobsubLite=?=true && STRINGLISTIMEMBER("fermilab", SupportedVOList) && regexp(".*dev.*", Machine) && InDownTime != true`
and of course is now working for me if I run it standalone...

Reply:
Sorry Marc - I'm a bit confused.  Am I understanding correctly that if you run the test by itself, it works, but if you run it as a part of the other tests, it fails?

Reply:
At the time of that writing, yes.  Then it failed on fifeutilgpvm03, but passed on my desktop.  Now it's running clean again on fifeutilgpvm03.   All without changing the code, or as far as I know, any of the other related software... So ... gremlins(?) aliens(?) nyetworking(?).